packages/frontend/README.md

```diff
@@ -2,25 +2,9 @@
 
 ## Development
 
-This project assumes that you have a local Rafiki backend instance up and running. Typically, both `backend` and this project would be used within the local Rafiki playground. See the [local environment setup](../../localenv) for more information on how to set it up.
+This project assumes that you have a local Rafiki backend instance up and running as well as a Kratos identity service and Mailslurper. Typically, all of these services, along with this project, would be used within the local Rafiki playground. See the [local environment setup](../../localenv) for more information on how to set it up.
 
-To start the project in development mode, we first need to generate the GraphQL types for our queries and mutations.
-
-To generate the GraphQL types file run:
-
-```sh
-pnpm --filter backend generate
-```
-
-To start the application run:
-
-```sh
-pnpm --filter frontend dev
-```
-
-Now you can access the application on [http://localhost:3005](http://localhost:3005).
-
-> NOTE: When running Rafiki Admin in the development environment, it will connect to Cloud Nine Wallet Admin GraphQL - [http://localhost:3001/graphql](http://localhost:3001/graphql).
+The Admin UI runs in development mode in the local playground, so when you have it running, you can see live code changes by simply refreshing your page.
```

Comment:
should we keep the links pointing to the admin pages here?

Reply:
The http://localhost:3005 link is no longer in use and doesn't work anymore. 
I can include the http://localhost:3001/graphql link but then why just that one, why not the rest of them - which is why I just advise people to to the localenv readme for that info.